packages/vite-plugin-cloudflare/playground/node-compat/__tests__/worker-postgres/postgres.spec.ts

```diff
@@ -12,6 +12,6 @@ test.runIf(!process.env.CI)(
 	"should be able to use pg library to send a query",
 	async () => {
 		const result = await getJsonResponse("/send-query");
-		expect(result!.id).toEqual("1");
+		expect(result!.id).toEqual("21");
```

Comment:
This test was consistently failing with this result so I assume the data in the database has changed.